src/sentry/static/sentry/app/components/charts/baseChart.tsx

```diff
@@ -493,7 +494,7 @@ const ChartContainer = styled('div')`
   }
 `;
 
-const BaseChartRef = React.forwardRef<ReactEchartsCore, Props>((props, ref) => (
+const BaseChartRef = React.forwardRef<ReactEchartsRef, Props>((props, ref) => (
```

Comment:
Not 100%, but I think changing the ref type here is right, because the `getEchartsInstance` function is declared on the class (and is missing from the proptypes that ReactEchartsCore exposes). Can someone with a bit more React xp let me know?

Reply:
Looks right to me as ref's don't get passed components.